make/boot.py

```
@@ -192,22 +213,349 @@ def command(args: Namespace) -> None:
+                    "--kernel-commandline=console=ttyS0,115200",
+                ]
+            ),
+        ):
+            print("boot: os install failed", file=sys.stderr)
+            error_exit(proc, cidfile)
+
+        if not stop(proc):
+            print("boot: install shutdown failed", file=sys.stderr)
+            error_exit(proc, cidfile)
+
+    except KeyboardInterrupt, Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)